src/fields/field_python.impl.hh

```diff
@@ -131,15 +125,12 @@ void FieldPython<spacedim, Value>::set_func(const string &func_name)
         PyTuple_SetItem(p_args_, i, p_value_);
     }
     p_value_ = PyObject_CallObject(p_func_, p_args_);
-
-    if (p_value_ == NULL) {
-        PyErr_Print();
-        // TODO: use cout to output also field arguments
-        xprintf(Err,"Failed to call field '%s' from the python module: %s\n", func_name.c_str(), PyModule_GetName(p_module_) );
-    }
+    PythonLoader::check_error();
 
     if ( ! PyTuple_Check( p_value_)) {
-        xprintf(UsrErr, "Field '%s' from the python module: %s doesn't return Tuple.\n", func_name.c_str(), PyModule_GetName(p_module_) );
+    	stringstream ss;
```

Comment:
This error is not an exception raised in Python so it should not be reported through the PythonLoader exception. 
Just use ExcMessage.